snuba/datasets/entity.py

```diff
@@ -222,7 +222,7 @@ def get_function_call_validators(self) -> Mapping[str, FunctionCallValidator]:
         """
         return {}
 
-    def get_writable_storage(self) -> WritableTableStorage:
+    def get_writable_storage(self) -> Optional[WritableTableStorage]:
```

Comment:
If the issue is only outcomes_raw. Why cannot we add the writable storage there ?

Reply:
Also discover